### ISSUE TITLE
Unable to right-click on accessibility check marks in diff view

### ISSUE THREAD

**Issue by EthyMoney** [desktop/desktop#19302]
### The problem

When in the diff view of a file, you can normally right-click on/near the line numbers to discard changes for a line or group of lines. Since the introduction of the new accessibility feature "Show check marks in the diff" was added, you cannot right-click over the check marks. 

You now have to right-click in an area around them for it to work, and right-clicking directly on the check marks does not do anything. This also makes it really difficult to right click on single line diffs, as the check mark takes up most of the clickable space. See the gif below for a demo of the problem.

### Release version

3.4.5

### Operating system

Windows 11 Pro Version 23H2

### Steps to reproduce the behavior

1. Turn on the "Show check marks in the diff" accessibility feature in the GitHub Desktop settings.
2. Make changes on a tracked file.
3. Open file in diff view in GitHub Desktop.
4. Attempt to right-click on the check marks to discard line changes.
5. Notice that the normal popup to discard line(s) does not appear.

### Log files

[2024-09-23.desktop.production.log](https://github.com/user-attachments/files/17104306/2024-09-23.desktop.production.log)

### Screenshots

![github-desktop-3-4-5](https://github.com/user-attachments/assets/3d820a0a-3d01-4422-859c-8948b59a448a)

### Additional context

Occurs in both unified and split diff display modes.

**Comment by sergiou87** [comment]
Thanks for your report!! I have submitted a fix for this in #19304
